core-bundle/src/Routing/Content/StringResolver.php

```diff
@@ -36,6 +36,10 @@ public function resolve(object $content): ContentUrlResult|null
 
         $url = $this->insertTagParser->replaceInline($content->value);
 
+        if ($url === '') {
+            return null;
```

Comment:
But doesn’t an empty string mean linking to the home page? At least this was how it worked in Contao 4 I think.

```suggestion
            $url = '/';
```

Reply:
In the `nav_default` Tempalte we use `./` - not sure if that makes a difference here.

Reply:
I don’t think `./` will work here because of how the UrlHelper works.